### DISCUSSION TITLE
Can ingested data in Seq be changed?

### DISCUSSION THREAD

**Discussion by tobias-graf-p** [datalust/seq-tickets#1837] [Help] [answered]
I would like to rename some keys of already ingested data in Seq. Can data which was already ingested in Seq later be changed? If yes, how?

**Comment by KodrAus** [comment, marked as answer]
Hi @tobias-graf-p :wave: Once logs are ingested into Seq they become fully immutable. With some effort you can replace data if you:

1. Formulate a query that includes the data you want to modify and export it as JSON
2. Use that same query to issue a manual delete of that data
3. Update the information you want in the exported JSON
4. Re-ingest the data

Depending on how much data there is this may or may not be feasible.

**Reply by tobias-graf-p** [reply]
OK, I understand. Thank you for your quick reply.
